Backend/test-database.py

```diff
@@ -8,124 +8,73 @@
-#                     "1/100", "0.05", "W8oSod", "7V", "5S5", "2S023", ".png")
-
-
 def insert_Drivetrain_data(Mechanism, NumOfPartsNeededToBeMachined, SizeOfPart, TypeofStock, ThicknessExtrusion, ThicknessPlate, MaterialPlate, LengthInInches, PartNumber, Date, Photo):
     request = f"INSERT INTO partSubmission.Drivetrain(Drivetrain_Mechanism, Drivetrain_NumofPartsNeedToBeMachined, Drivetrain_SizeOfPart, Drivetrain_TypeofStock, Drivetrain_ThicknessExtrusion, Drivetrain_ThicknessPlate, Drivetrain_MaterialPlate, Drivetrains_LengthInInches, Drivetrain_PartNumber, Drivetrain_Date, Drivetrain_Photo) VALUES('{Mechanism}', '{NumOfPartsNeededToBeMachined}', '{SizeOfPart}','{TypeofStock}', '{ThicknessExtrusion}', '{ThicknessPlate}', '{MaterialPlate}', '{LengthInInches}', '{PartNumber}', '{Date}', '{Photo}');"
     mycursor.execute(request)
     mydb.commit()
     rows = mycursor.fetchall()
     print(rows)
 
-
-# insert_Drivetrain_data("Foreba9r", "555009", "01/8", "Mo^ney",
-#                        "1/100", "0.0005", "W8oSod", "7V", "5S5", "2S023", ".png")
-
-
 def insert_Elevator_data(Mechanism, NumOfPartsNeededToBeMachined, SizeOfPart, TypeofStock, ThicknessExtrusion, ThicknessPlate, MaterialPlate, LengthInInches, PartNumber, Date, Photo):
     request = f"INSERT INTO partSubmission.Elevator (Elevator_Mechanism, Elevator_NumofPartsNeedMachined, Elevator_SizeOfPart, Elevator_TypeOfStock, Elevator_ThicknessExtrusion, Elevator_ThicknessPlate, Elevator_MaterialPlate, Elevator_LengthInInches, Elevator_PartNumber, Elevator_Date, Elevator_Photos) VALUES('{Mechanism}', '{NumOfPartsNeededToBeMachined}', '{SizeOfPart}','{TypeofStock}', '{ThicknessExtrusion}', '{ThicknessPlate}', '{MaterialPlate}', '{LengthInInches}', '{PartNumber}', '{Date}', '{Photo}');"
     mycursor.execute(request)
     mydb.commit()
     rows = mycursor.fetchall()
     print(rows)
 
-
-# insert_Elevator_data("Foreba9r", "555009", "01/8", "Mo^ney",
-#                      "1/100", "0.05", "W8612oSod", "7V", "5S5", "2S023", ".png")
-
-
 def insert_Forebar_data(Mechanism, NumOfPartsNeededToBeMachined, SizeOfPart, TypeofStock, ThicknessExtrusion, ThicknessPlate, MaterialPlate, LengthInInches, PartNumber, Date, Photo):
     request = f"INSERT INTO partSubmission.Forebar (Forebar_Mechanism, Forebar_NumofPartsNeedMachined, Forebar_SizeOfPart, Forebar_TypeOfStock, Forebar_ThicknessExtrusion, Forebar_ThicknessPlate, Forebar_MaterialPlate, Forebar_LengthInInches, Forebar_PartNumber, Forebar_Date, Forebar_Photos) VALUES('{Mechanism}', '{NumOfPartsNeededToBeMachined}', '{SizeOfPart}', '{TypeofStock}','{ThicknessExtrusion}', '{ThicknessPlate}', '{MaterialPlate}', '{LengthInInches}', '{PartNumber}', '{Date}', '{Photo}');"
     mycursor.execute(request)
     mydb.commit()
     rows = mycursor.fetchall()
     print(rows)
 
-
-# insert_Forebar_data("Foreba9r", "555009", "01/8", "Mo^ney",
-#                     "1/100", "0.05", "W8oSod", "7V", "5S5", "2S023", ".png")
-
-
```

Comment:
I personally would recommend leaving the comments in this file - these can be good references for how the different functions work, and how you can use them elsewhere in the app.

As a good rule of thumb, I usually remove commented-out code if I don't plan to use it in the future when working on apps. This file is a bit special as we don't actually use it in our app - it's more of a playground for us to see how things work and test functions before we move them to the app.

Some comments I also add in when I want to make something more clear - comments that explain why something is implemented a specific way, or something important I learned when writing the code are super helpful to leave in for the future as well